src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -10,7 +10,12 @@ import {
   PageSection,
 } from '@patternfly/react-core';
 import { WizardStepType } from '@patternfly/react-core/dist/esm/components/Wizard';
-import { useFlag } from '@unleash/proxy-client-react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Why do we need to ignore this?

Reply:
I wonder if importing this from a relative path would fix this and get rid of the need for the `ts-ignore`